### PR TITLE
Add bazel module for Universal_Robots_Client_Library version 1.4.0

### PR DESCRIPTION
**Pull request by b-bremer**
Adds BUILD targets for 

* filegroup for `rtde recipes`
* The `ur_client_library`
* example binary `full_driver`
* tests that don't depend on a simulator or real robot. (they however share ports)

I could unfortunately not find a stable release url. @bazel-io skip_check unstable_url

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (universal-robots-client-library) have been updated in this PR. Please review the changes.

**Comment by udaya2899** [comment]
Nit: Title is `bazel`, not `blaze`

**Comment by b-bremer** [comment]
@bazel-io skip_check unstable_url because I can unfortunately not find a stable release url.

**Comment by b-bremer** [comment]
> @bazel-io skip_check unstable_url because I can unfortunately not find a stable release url.

The recommendation by https://buildkite.com/bazel/bcr-presubmit/builds/8927#01934a88-286b-4685-bb75-41df0c95a7e1 was to add that comment. 

The release page (https://github.com/UniversalRobots/Universal_Robots_Client_Library/releases/tag/1.4.0) only lists the link I used in this request.

image

File: modules/universal-robots-client-library/1.4.0/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
++)
++
++exports_files([
++    "LICENSE_apache_2_0",
++])
++
++license(
++    name = "license",
++    package_name = "universal-robots-client-library",
++    license_kinds = [
```
Comment (udaya2899) [inline modules/universal-robots-client-library/1.4.0/patches/add_build_file.patch:20]: Add `license_text = "LICENSE_apache_2_0"` to fix presubmits

**Review by Wyverald — APPROVED** [review]

**Pull request merged**